doc/source/dev/governance/governance.rst

```diff
@@ -378,6 +378,22 @@ A list of current Institutional Partners is maintained at the page
 :ref:`governance-people`.
 
 
+Commit Rights in the NumPy Code Base
+====================================
+
+The NumPy Steering Council can grant and revoke commit rights to the NumPy code
+repository, currently hosted at `github`_.
+
+The policy for granting commit priveleges is based on trust, with the
+understanding that they are needed for many of the mundane tasks required
+to maintain NumPy. New people should be conservative in exercising their
+commit bit, at the beginning using them to triage issues, add labels, and
```

Comment:
I think I would avoid the "bit" part, its a prettty technical name. Personally, do not like "people" much, too (also maybe mundane). The last sentence is probably unnecessary in this document?
Anyway, Nathaniel, etc. are better at this stuff, so just a quick procrastination comment.